### PR TITLE
Feature/redesign

### PR DESCRIPTION
**Pull request by andriikamaldinov1**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/JsDaddy/ngx-loader-indicator/blob/develop/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/app/shared/form/form.component.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<form (ngSubmit)="login()" [formGroup]="form" [jsdaddyLoader]="isLoading" class="column">
+    <ng-container *ngFor="let formItem of formItems">
```
Comment (GlebChiz) [inline src/app/shared/form/form.component.html:2]: trakBy

File: src/app/cards/cards.component.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+export class CardsComponent implements AfterViewInit {
+    @Input() public cardDocs!: ICard[];
+
+    @ViewChildren('cards') public cards!: QueryList<ElementRef>;
+
+    public readonly activeCardId$: Observable<number> = inject(ScrollService).activeCard$;
+    public readonly openSourcePath = OpenSourcePath.OPEN_SOURCE;
+    public readonly trackByPath = inject(TrackByService).trackBy('id');
+    public readonly cardTypeInput: CardType = CardType.INPUT;
+    public readonly cardTypeLoader: CardType = CardType.LOADER;
```
Comment (xakeppok) [inline src/app/cards/cards.component.ts:54]: you have only 2 states. for what are you using enum?

File: src/app/cards/cards.component.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        AssetPipe,
+        CustomLoaderComponent,
+        FormComponent,
+        InputComponent,
+    ],
+    providers: [ScrollService],
+    templateUrl: './cards.component.html',
+    styleUrls: ['./cards.component.scss'],
+})
+export class CardsComponent implements AfterViewInit {
```
Comment (xakeppok) [inline src/app/cards/cards.component.ts:45]: as i see we have it in the all open source. is it make sense to make it reusable?

File: src/index.html [hunk 4]
```diff
@@ -2,11 +2,14 @@
...
+        <title>Ngx Loader Indicator | JsDaddy</title>
         <base href="/" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta
+            name="description"
+            content="Awesome loader for angular applications. No wrappers only you elements" />
     </head>
     <body>
-        <ngx-loader-indicator-demo-root></ngx-loader-indicator-demo-root>
+        <jsdaddy-open-source-root></jsdaddy-open-source-root>
```
Comment (GlebChiz) [inline src/index.html:13]: `<jsdaddy-open-source-root></jsdaddy-open-source-root> -> <jsdaddy-open-source-root />`

File: src/main.ts [hunk 5]
```diff
@@ -14,6 +19,14 @@ if (environment.production) {
 bootstrapApplication(AppComponent, {
     providers: [
         provideAnimations(),
+        provideRouter([]),
```
Comment (xakeppok) [inline src/main.ts:22]: is this application use routing? there are only 1 page

File: src/app/cards/cards.component.ts (hunk 3 above)
Comment (andriikamaldinov1) [reply src/app/cards/cards.component.ts:45]: interfaces are different

**Pull request merged**